cms/static/sass/course-unit-mfe-iframe-bundle.scss

```diff
@@ -0,0 +1,645 @@
+@import 'cms/theme/variables-v1';
+@import 'elements/course-unit-mfe-iframe';
+
+.wrapper-content.wrapper {
+  padding: 5px;
+}
+
+body .wrapper-xblock {
+  background-color: $transparent;
+  border-radius: 6px;
+  border: none;
+
+  &:hover {
+    border-color: none;
+  }
+
+  .xblock-header-primary {
+    padding: 24px;
+    padding-bottom: 12px;
+    border-bottom: none !important;
+
+    .header-details .xblock-display-name {
+      font-size: 22px;
+      line-height: 28px;
+      font-weight: 700;
+      color: $black;
+    }
+  }
+
+  &.level-element {
+    box-shadow: 0 2px 4px rgba(0, 0, 0, .15), 0 2px 8px rgba(0, 0, 0, .15);
+    margin: 0 0 28px 0;
+  }
+
+  &.level-element .xblock-header-primary {
+    background-color: $white;
+  }
+
+  &.level-element .xblock-render {
+    background: $white;
+    margin: 0;
+    padding: 24px;
+    padding-top: 12px;
+  }
+
+  .wrapper-xblock .header-actions .actions-list .action-item .action-button {
+    color: $black;
+
+    @extend %button-styles;
```

Comment:
[nit] Let's declare all extended rules at the top of the block for separation and simple overriding.

Reply:
Moved at the top